content/community/governance/index.md

```diff
@@ -51,9 +56,9 @@ The [Project Management Office (PMO)](/community/pmo) supports the different bod
 
 **Proposed governance workflow:**
 
-* [Galaxy Exec](/community/exec/): group of PIs including Frederik Coppens, Björn Grüning, Ross Lazarus, Andrew Lonie, Anton Nekrutenko, and Michael Schatz.
-* [Global Steering Committee](/community/steering/): collaborators from active Galaxy communities.
-* [Working groups](/community/wg/): groups of community members responsible for various parts of the Galaxy ecosystem.  
+* [Galaxy Executive Board](/community/governance/geb): group of PIs including Frederik Coppens, Björn Grüning, Ross Lazarus, Andrew Lonie, Anton Nekrutenko, and Michael Schatz.
+* [Galaxy Community Board](/community/governance/gcb): collaborators from active Galaxy communities.
+* [Galaxy Technical Board](/community/governance/gtb): groups of community members responsible for various parts of the Galaxy ecosystem.
 * Idea by Björn Grüning, Hans-Rudolf Hotz and others.
 
 The overall idea is:
```

Comment:
You reference the Steering committee in the next line.
Should be more like: 

In addition to their self-governance and collaboration, these boards interact with each other to build the annual Galaxy roadmap (link):

1. The *Technical Board* and the *Community Board* are periodically asked for high-level input, representing the interests of the wider Galaxy community.
2. The *Galaxy Executive Board* reviews suggestions and creates a final high-level roadmap (1 week). 
3. This roadmap is then given to the *Technical Board* which will (3 weeks):
 * Add technical details.
 * Estimate the timings.
 * Add cross working group dependencies.
 * Add other requirements (e.g. hardware).
 * A detailed roadmap is given to the *Galaxy Executive Board* for the final decision (1 week).

3. The *Galaxy Executive Board* signs off the roadmap and publishes it on GitHub.

Reply:
also link the 'roadmap' page here

Reply:
I see. This section needs a new image and then a text that fits the image. If anyone wants to give it a go.

Reply:
I took that over as well, but we need a new image.